test/y2storage/clients/partitions_proposal_test.rb

```diff
@@ -189,6 +189,29 @@
         expect(proposal["warning_level"]).to be :blocker
       end
     end
+
+    context "in simple mode" do
+
+      it "returns a label_proposal with 'Default'" do
+        proposal = subject.make_proposal("simple_mode" => true)
+        expect(proposal).to be_a Hash
+        expect(proposal).to include("label_proposal")
+        expect(proposal["label_proposal"]).to be_a Array
+        expect(proposal["label_proposal"]).to eq ["Default"]
+      end
+
+      it "returns a label_proposal with 'Custom' after using the partitioner" do
+        subject.make_proposal("simple_mode" => true)
```

Comment:
I guess this first call to `#make_proposal` is not necessary, isn't it?

Reply:
Since it does that normally before a user can enter the expert partitioner, this makes sure that everything is properly initialized. It might work without this, but it's more realistic to have it IMHO.